ci/Dockerfile.aleph-node

```
@@ -0,0 +1,26 @@
+FROM public.ecr.aws/p6e8q1z1/aleph-node:r-10.0
+
+# Save node and sudo account ids for convenience
+RUN aleph-node key inspect //Alice | grep "SS58 Address" | awk "{print \$3}" \
+  > /node/sudo_account_id
+RUN aleph-node key inspect //1 | grep "SS58 Address" | awk "{print \$3}" \
+  > /node/node_account_id
```

IIUC, the advise for writing leaner docker containers is to batch commands together if they are not expected to be updated independently:
```suggestion
RUN aleph-node key inspect //Alice | grep "SS58 Address" | awk "{print \$3}" > /node/sudo_account_id \
      && aleph-node key inspect //1 | grep "SS58 Address" | awk "{print \$3}" > /node/node_account_id
```

---

The things below could probably be also included in that single RUN command. Unless you think you might need to change them w/o changing key generation (which I don't think is the case).

Fewer layers == smaller docker containers.

---

:+1: will tweak that